scripts/exglobal_diag.sh

```diff
@@ -87,11 +87,11 @@ lrun_subdirs=${lrun_subdirs:-".true."}
 
 ################################################################################
 # If requested, generate diagnostic files
-if [ $GENDIAG = "YES" ] ; then
-   if [ $lrun_subdirs = ".true." ] ; then
-      for pe in $DIAG_DIR/dir.*; do
-         pedir="$(basename -- $pe)"
-         $NLN $pe $DATA/$pedir
+if [[ "${GENDIAG}" = "YES" ]] ; then
+   if [[ "${lrun_subdirs}" = ".true." ]] ; then
+      for pe in "${DIAG_DIR}"/dir.*; do
+         pedir=$(basename -- "${pe}")
+         "${NLN}" "${pe}" "${DATA}/${pedir}"
```

Comment:
This is what is causing CI to fail. Can't quote variables when we use them as commands, because it treats the whole thing (here `/bin/ln -sf`) as a single command.
```suggestion
         ${NLN} "${pe}" "${DATA}/${pedir}"
```

Reply:
Good catch. Thank you.